src/clevertap.js

```diff
@@ -143,6 +143,27 @@ export default class CleverTap {
       return this.#device.getGuid()
     }
 
+    this.getAccountID = () => {
+      return this.#account.id
+    }
+
+    this.getDCDomain = () => {
+      return 'dc-' + this.#account.finalTargetDomain
+    }
+
+    // Get the Direct Call sdk version
+    this.setDCSDKVersion = (ver) => {
```

Comment:
@KambleSonam The function name says "set" but the comment above it says "get". Can we add a more descriptive comment please?